meta/runtime.yml

```diff
@@ -156,7 +157,7 @@ plugin_routing:
     foreman_user:
       redirect: theforeman.foreman.user
     foreman_usergroup:
-      redirect: theforeman.foreman.usergroup
+      redirect: theforeman.foreman.wait_for_task
```

Comment:
that seems wrong :)

Reply:
Corrected

Reply:
technically, that is correct, a new module doesn't need a mapping entry for the old name.

Reply:
@evgeni Yes, it has been totally removed in my latest push :smiley: